packages/lab/src/stepper-input/StepperInput.tsx

```diff
@@ -11,7 +19,7 @@ import stepperInputCss from "./StepperInput.css";
 const withBaseName = makePrefixer("saltStepperInput");
 
 export interface StepperInputProps
-  extends Omit<InputProps, "onChange" | "emptyReadOnlyMarker"> {
+  extends Omit<InputProps, "emptyReadOnlyMarker"> {
```

Comment:
What will be the `onChange` event if users uses it? Will they be confused with `onValueChange`?

Reply:
It would be passed to the input, it could be simpler to exclude it though